generated/nirfmxinstr/nirfmxinstr_service.cpp

```diff
@@ -26,8 +26,12 @@ namespace nirfmxinstr_grpc {
   NiRFmxInstrService::NiRFmxInstrService(
       NiRFmxInstrLibraryInterface* library,
       ResourceRepositorySharedPtr session_repository, 
+      ViSessionResourceRepositorySharedPtr vi_session_resource_repository,
```

Comment:
RFmxInstr is a good example of the new C++ code that (a) passes through the repository pointer and (b) implements the access/init methods.